vault/templates/install.sh.tpl

```diff
@@ -15,7 +15,3 @@ sudo chown root:root /usr/local/bin/vault
 sudo /tmp/teleport/install
 
 sudo systemctl daemon-reload
```

Comment:
Maybe move this too?
not sure if the systemd files will be written before the command, but after the files are in place a daemon-reload is always needed.

Reply:
Files are written before `runcmd` and also before the standalone scripts are run, so at this point all systemd files will already be in place. And this is also needed before the teleport bootstrap script is run, so I think we should keep it here.

Reply:
can you move the reload to runcmd? so its all together

Reply:
that's what I meant, it won't work if I move it to `runcmd`, because the `daemon-reload` is needed by the teleport bootstrap script, which is run after the `install.sh` but before `runcmd`.

Reply:
ah ok.